### PR TITLE
Remove new navigation switch

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

**Header**
Delete old style entry points (previously used if switch OFF):
- `static/src/stylesheets/head.amp.scss` (superceded by `head.amp-new.scss`)
- `static/src/stylesheets/head.amp-liveblog.scss` (superseded by `head.amp-liveblog-new.scss`)
- `static/src/stylesheets/head.identity.scss` (superseded by `head.new-identity.scss`)

Remove navigation styles separately inlined in head (now inlined as part of `head.common.scss`):
- `head.navigation.scss`
- `head.new-navigation.scss`

Reinstate navigation styles in `head.common.scss` by uncommenting link to `layout/_navigation.scss`
Delete old styles in `static/src/stylesheets/layout/navigation/*` directory
Delete old AMP styles in `static/src/stylesheets/amp/nav/*` directory

**Footer**
Remove entry points for the separate `footer.css` file initially created when switch implemented (now included in `common.garnett.scss`):
- `static/src/stylesheets/footer.scss`
- `static/src/stylesheets/new-footer.scss`

Reinstate footer styles in `_common.garnett.scss` by uncommenting link to `layout/_footer.scss`
Delete old styles in `static/src/stylesheets/layout/footer/*` directory
Delete old AMP styles in `static/src/stylesheets/amp/footer/*` directory

**Templates**
Remove `NewNavigationEnabled` switch

**Styles**
Remove `.has-new-navigation` override

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Review by NataliaLKB — APPROVED** [review]
Great cleanup! Thanks for doing this. It may be easier to do in a subsequent PR, but could we remove the `new` prefix to the file names?

**Comment by NataliaLKB** [comment]
Ah never mind, you were going to do that anyway! 🎉

File: admin/app/tools/CloudWatch.scala [hunk 1]
```diff
@@ -90,7 +90,7 @@ object CloudWatch extends Logging {
     "head.content.css",
     "head.facia.css",
     "head.football.css",
-    "head.identity.css",
```
Comment (walaura) [inline admin/app/tools/CloudWatch.scala]: out of curiosity, what is this list used for?

**Review by walaura — APPROVED** [review]
thanks for the identity stuff!! did you get the chance to test profile pages like mma in CODE?

**Comment by GHaberis** [comment]
@walaura I tested some identity pages on CODE. Not sure what 'mma' is though? Happy to stick it on CODE again to give profile pages another look?

File: admin/app/tools/CloudWatch.scala (hunk 1 above)
Comment (GHaberis) [reply admin/app/tools/CloudWatch.scala]: It looks like it's used for tracking bundle sizes in Frontend Admin dashboards.

**Pull request merged**